client/deis.py

```diff
@@ -316,13 +316,14 @@ def readable_datetime(datetime_str):
             return 'Just now'
         else:
             return "{}{}ago".format(hour_str, min_str)
+
     # if it happened yesterday, say "yesterday at 3:23 pm"
-    yesterday = now + relativedelta.relativedelta(days=-1)
-    if delta.days <= 2 and dt.day == yesterday.day:
```

Comment:
Ah, I see your point. I thought that r.days would have been 31 in that case, but it's only showing as 1, causing it to show "Yesterday" every month interval. :+1: